source/includes/agents/_50-bulk-update.md.erb

```diff
@@ -0,0 +1,53 @@
+## Bulk Update agents
+
+```shell
+curl 'https://ci.example.com/go/api/agents' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v2+json' \
+      -H 'Content-Type: application/json' \
+      -X PATCH \
+      -d '{
+            "uuids": [
+              "adb9540a-b954-4571-9d9b-2f330739d4da",
+              "adb528b2-b954-1234-9d9b-b27ag4h568j1"
+            ],
+            "operations": {
+              "environments": {
+                "add": ["Dev", "Test"],
+                "remove": ["Production"]
+              },
+              "resources": {
+                "add": ["Linux", "Firefox"],
+                "remove": ["Crome"]
+              }
+            },
+            "agent_config_state" : "enabled"
+          }'
```

Comment:
Could you add a quick section about what these attributes mean? Something along the lines of https://api.go.cd/current/#update-an-agent